client/src/components/Tickets/TicketCreateForm.js

```diff
@@ -188,6 +196,14 @@ function TicketCreateForm({
                       />
                     </Col>
                   </Row>
+                  {!isAuthenticated && (
```

Comment:
If the captcha fails, do we show an error?

Reply:
Should the ReCAPTCHA be placed into its own `_common` component where the ref, rechaptcha response state, error state are encapsulated for reuse in other places.  I'm wondering about re-use in the request access page (in @jmartinez-tacc 's PR at the moment)or in other future places.  Or maybe that would be best as a follow-on JIRA task after this PR?  @rstijerina @williamkwon97

Reply:
@nathanfranklin I think that's a good idea, and a good opportunity for a follow-on ticket

Reply:
@nathanfranklin @rstijerina  a new ticket has been created https://jira.tacc.utexas.edu/browse/FP-1344

Reply:
@williamkwon97 Do we capture and display an error if one occurs on the backend here?

Reply:
![Screen Shot 2021-11-16 at 12 32 11 PM](https://user-images.githubusercontent.com/34716439/142044736-a034960f-f7ba-44bc-a26b-f79295252b33.png)
yes, it displays an error message.